index.html

```diff
@@ -105,6 +107,29 @@ <h3>
           reaches its maximum capacity and gets flushed.
         </p>
       </dd>
+      <dt>
+        <dfn><code>jitterBufferReceivedPackets</code></dfn> of type
+        <span class="idlMemberType"><a href="https://heycam.github.io/webidl/#idl-unsigned-long-long">unsigned long long</a></span>
+      </dt>
+      <dd>
+        <p>
+          Experimental stat which will be available under origin trial in
+          Chrome. Counts the total number of packets received by the jitter
+          buffer.
```

Comment:
This is packets, not samples, so this is not the same as totalSamplesReceived, correct?

Is there a difference between packetsReceived (https://w3c.github.io/webrtc-stats/#dom-rtcreceivedrtpstreamstats-packetsreceived) and jitterBufferReceivedPackets? Packets can get dropped before entering the jitter buffer?

Reply:
Correct.

They are not exactly the same for various implementation specific reasons (e.g. we have not started playing yet, the payload type is unknown). But I think using packetsReceived should be fine actually (after looking more closely on how it is reported) and any discrepancy between it and jitterBufferPacketsReceived would just drag down the average relativePacketArrivalDelay slightly when used as a denominator.

Will update the PR.